lib/chef-dk/policyfile_services/export_repo.rb

```diff
@@ -85,13 +109,35 @@ def export
 
       private
 
-      def create_repo_structure
-        FileUtils.rm_rf(cookbooks_dir)
-        FileUtils.rm_rf(policyfiles_data_bag_dir)
+      def with_staging_dir
+        p = Process.pid
+        t = Time.new.utc.strftime("%Y%m%d%H%M%S")
+        Dir.mktmpdir("chefdk-export-#{p}-#{t}") do |d|
```

Comment:
I'm trying to think back to the old times before we fixed all this in Chef. IIRC, both windows and selinux had similar issues, where files created in tmp would have severely restricted permissions, and then we would `mv` them to the final location, which would not update the permissions and everything would be borked. What we're doing here is copying the cookbooks to this tempdir, and creating other stuff with `File.open(item_path, "wb+")`. I _think_ this means it will get the permissions of a newly created file? Then we `mv` it at the end, so it will keep the permissions it had. I'm trying to figure out if this will cause a problem if you were to try to run chef-client local mode on the same box after running `chef export`.